diff --git a/debian/scripts/find-firmware.pl b/debian/scripts/find-firmware.pl
index b24e66e..5a7051d 100755
--- a/debian/scripts/find-firmware.pl
+++ b/debian/scripts/find-firmware.pl
@@ -8,7 +8,8 @@ die "no directory to scan" if !$dir;
 
 die "no such directory" if ! -d $dir;
 
-die "strange directory name" if $dir !~ m|^(.*/)?(5.3.\d+\-\d+\-pve)(/+)?$|;
+# Enable custom kernel names - not just "-pve"
+# die "strange directory name" if $dir !~ m|^(.*/)?(5.3.\d+\-\d+\-pve)(/+)?$|;
 
 my $apiver = $2;
 
