diff --git a/debian/scripts/find-firmware.pl b/debian/scripts/find-firmware.pl
index 9492bc9..4ef6b2f 100755
--- a/debian/scripts/find-firmware.pl
+++ b/debian/scripts/find-firmware.pl
@@ -8,6 +8,7 @@ die "no directory to scan" if !$dir;

 die "no such directory" if ! -d $dir;

-die "strange directory name: $dir" if $dir !~ m|^(.*/)?(5.\d.\d+\-\d+\-pve)(/+)?$|;
+# Enable custom kernel names - not just "-pve"
+# die "strange directory name: $dir" if $dir !~ m|^(.*/)?(5.\d.\d+\-\d+\-pve)(/+)?$|;

 my $apiver = $2;
