diff --git a/drivers/iommu/intel-iommu.c b/drivers/iommu/intel-iommu.c
index f83a9a302f8e..ee0be8e2b055 100644
--- a/drivers/iommu/intel-iommu.c
+++ b/drivers/iommu/intel-iommu.c
@@ -5112,8 +5112,7 @@ static int intel_iommu_attach_device(struct iommu_domain *domain,
 
 	if (domain->type == IOMMU_DOMAIN_UNMANAGED &&
 	    device_is_rmrr_locked(dev)) {
-		dev_warn(dev, "Device is ineligible for IOMMU domain attach due to platform RMRR requirement.  Contact your platform vendor.\n");
-		return -EPERM;
+		dev_warn(dev, "Device is not suitable for IOMMU domain attach due to platform RMRR requirement. Bugs might occour as this restriction has been bypassed.\n");
 	}
 
 	if (is_aux_domain(dev, domain))
